maas-schemas/schemas/maas-backend/regions/regions-options/request.json

```diff
@@ -3,13 +3,9 @@
   "description": "Request schema for regions-options",
   "type": "object",
   "properties": {
-    "identityId": {
-      "$ref": "https://schemas.maas.global/core/components/units.json#/definitions/identityId"
-    },
     "headers": {
       "$ref": "https://schemas.maas.global/core/components/api-common.json#/definitions/headers"
     }
   },
-  "required": ["identityId"],
```

Comment:
Just removing the required should be enough I think

Reply:
Correct but then again why leave `identityId` if you don't need it at all? Thus, I removed that too.

Reply:
Ah, well if you remove identityId and `additionalProperties` false then it would be failing for older clients?

Reply:
How is that? Older client should use previous schema, i.e. < 15.11.0

Reply:
The schema is for backend right?
Meaning when we update/release backend this would apply to every request to region option endpoint to our backend. And if there is no client force-update older client would still request region option with Auth header which translated into identityId, and would fail the schema validation

Reply:
Yes, I get your point. You're right and wrong :) The `additionalProperties` would fail the request just as you described. However, when you remove authorizer from endpoint then there won't be any `identityId` at all.

Reply:
if you don't believe me then check this yourself: GET https://api.boombox.development.maas.global/regions/options

Reply:
Ok got it.
Though the GET above is 403 😅

Reply:
You still need `X-API-Key` :D